app/components/Form/SelectProjectStatusWidget.tsx

```diff
@@ -68,12 +68,17 @@ const SelectProjectStatus: React.FunctionComponent<WidgetProps> = (props) => {
       </Dropdown>
       <style jsx>
         {`
-          :global(input) {
+          div :global(input) {
             width: 100%;
           }
```

Comment:
Looks like this did have a side effect: It needs to be added in `MoneyWidget` as well now, as it's not using the whole width anymore